pmlc/conversion/gpu/gpu_to_vulkan.cc

```diff
@@ -210,14 +235,27 @@ ConvertGpuLaunchFuncToVulkanCalls::bindBuffers(Location loc, OpBuilder &builder,
     Value descriptorBinding = builder.create<LLVM::ConstantOp>(
         loc, getLLVMInt32Type(), builder.getI32IntegerAttr(bindIndex));
     if (auto memRefType = buffer.getType().dyn_cast_or_null<MemRefType>()) {
+      auto shape = memRefType.getShape();
+      uint32_t numElement = 1;
+      for (auto dim : shape) {
+        numElement *= dim;
+      }
+
       auto elementType = memRefType.getElementType();
+      bufferElementTypes.push_back(elementType);
+      uint32_t elementTypeSize = elementType.getIntOrFloatBitWidth() / byteBits;
```

Comment:
Use `llvm::divideCeil` to account for `i1` types.

Reply:
It is a good point, thanks.